AutoFoldCode.py

```diff
@@ -1,6 +1,41 @@
 import os
 import sublime
 import sublime_plugin
+import zlib
+
+'''
+Storage file format (current format):
+
+  {
+    "<filename>":
+    {
+      "<content_crc32>":
+      [
+        [
+          <region_start>,
+          <region_end>
+        ],
+        ...
+      ],
+      ...
+    },
+    ...
+  }
+
+Storage file format (legacy format):
```

Comment:
We'll mark your change as `1.2.0`, so if you could put in the version numbers in the titles here that would be great.

For example:

```
Storage file format (1.2.0+):
...
Storage file format (pre-1.2.0):